platform-coop-toolkit/coop-map-directory-index
# feat: rebuild home page cards as Svelte components

## Description

Rebuilds home page cards as Svelte components. Rather than rebuilding the entire list of cards using string concatenation when the map view changes, the Svelte store for map features is updated and individual cards are added or removed from the card grid.

Updates serializers to provide more detailed/required information for user and organization endpoints to support card generation.

## Steps to test

1. Check out branch.
2. `npm ci && npm run dev`
3. Zoom and pan home page map view.

**Expected behavior:** Cards update as expected.

## Additional information

This PR does not add individuals to the map. Individuals were adding during development to test the functionality. That will be addressed in a separate PR.

## Related issues

Not applicable.
